plugins/producer_api_plugin/producer_api_plugin.cpp

```diff
@@ -104,8 +104,6 @@ void producer_api_plugin::plugin_startup() {
             INVOKE_R_V(producer, paused), 201),
```

Comment:
`resume` not safe when running read-only trxs as it `abort_block()` and `schedule_production_loop()`. For consistency I think moving `pause` to `general` would be good as well. Currently `pause` is safe, but if we ever decide to modify `pause` to `pause` the current block it could cause an issue.

Reply:
I thought the read only window could not be activated when nodeos is in block producing mode. Would the above still be called in non-producing mode?

Reply:
Sadly, yes. You can call `pause()/resume()` on non-producing nodes.